pkg/operator/ceph/csi/spec.go

```diff
@@ -243,6 +247,20 @@ func StartCSIDrivers(namespace string, clientset kubernetes.Interface, ver *vers
 		deployProvSTS = true
 	}
 
+	tp.ResizerImage = os.Getenv("ROOK_CSI_RESIZER_IMAGE")
+	if tp.ResizerImage == "" {
+		tp.ResizerImage = defaultResizerImage
+	}
+
+	if ver.Major == KubeMinMajor {
+		if ver.Minor < kubeMinVerForFilesystemRestore {
+			logger.Warning("CSI Filesystem volume expansion requires Kubernetes version >=1.15.0")
```

Comment:
You could have used `kubeMinVerForFilesystemRestore` in the message but this is not likely to change I guess...

Reply:
i think this should be okay, or else it looks like

 ```logger.Warning("CSI Filesystem volume expansion requires Kubernetes version >=1.%s.0",kubeMinVerForFilesystemRestore)```

Reply:
Yes, it's ok.